examples/oft-hyperliquid/contracts/MyHyperLiquidOFT.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+import { OFT } from "@layerzerolabs/oft-evm/contracts/OFT.sol";
+
+contract MyHyperLiquidOFT is OFT {
+    constructor(
+        string memory _name,
+        string memory _symbol,
+        address _lzEndpoint,
+        address _delegate
+    ) OFT(_name, _symbol, _lzEndpoint, _delegate) Ownable(_delegate) {}
+
+    function mint(address _to, uint256 _amount) public virtual {
```

Comment:
Do we always want to expose the mint/burn here? In our other examples, usually we only tell them to expose if absolutely needed or in the mock example for testing